project/components/resource_request/models.py

```diff
@@ -48,5 +50,7 @@ class ResourceRequest(DBModel):
     requested_for = Column(VARCHAR(length=256), nullable=False)
     requested_at = Column(TIMESTAMP(timezone=True), default=datetime.utcnow, nullable=False)
     completed_at = Column(TIMESTAMP(timezone=True), nullable=True)
+    message = Column(String(length=100), nullable=True)
```

Comment:
- Please use `VARCHAR` here for a better understanding that `varchar` data type will be used in db.
- I would suggest to increase length at least to 256 since 100 it might be not enough for a message.
```suggestion
    message = Column(VARCHAR(length=256), nullable=True)
```

Reply:
I've updated it, though from what I see it really makes no difference. I'm leaving the length as is as that comes from the requirements.